src/HotChocolate/Core/src/Execution/Batching/BatchExecutor.cs

```diff
@@ -27,13 +27,15 @@ internal partial class BatchExecutor
 
     public IAsyncEnumerable<IQueryResult> ExecuteAsync(
         IRequestExecutor requestExecutor,
-        IEnumerable<IQueryRequest> requestBatch)
+        IEnumerable<IQueryRequest> requestBatch,
+        bool allowParallelExecution)
     {
         return new BatchExecutorEnumerable(
             requestBatch,
             requestExecutor,
             _errorHandler,
             _typeConverter,
-            _inputFormatter);
+            _inputFormatter,
+            /*allowParallelExecution*/true); // TODO: How to set this from caller?
```

Comment:
I see traces of allowParallelExecution, but I don't see how to set it or use it. I believe this is possibly unused? It is also possible that I've started abusing something meant for something different